### DISCUSSION TITLE
ArgumentCountError Too few arguments to function Illuminate\Support\Str::replaceFirst()

### DISCUSSION THREAD

**Discussion by gianmarxWebmapp** [livewire/livewire#2448] [1. Help]
with composer self update he updated me to version 1.10.20 and now composer update goes to me

but there is a new error that I don't understand, related to how I recall the livewire view in the laravel view.

i get this error:

	ArgumentCountError
Too few arguments to function Illuminate\Support\Str::replaceFirst(), 2 passed in /Users/gianmarx/hoqu-laravel/vendor/livewire/livewire/src/helpers.php on line 12 and exactly 3 expected (View: /Users/gianmarx/hoqu-laravel/resources/views/dashboard.blade.php)

I recall the livewire view like this:

                @livewire('menu-dash-board')
and I get the above error

if I comment the page works

**Comment by PhiloNL** [comment]
I would recommend taking a look at the actual blade output fille in `storage/framework/views/*.blade.php`. Often the filename of this file is in your error stack trace. This might give you some more insights into what is causing the error.

**Comment by shibbirweb** [comment]
I solved this issue by running
` php artisan optimize:clear`
